File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
import sys
import math

a, b = list(map(int, sys.stdin.read().strip().split(' ')))

def lcm (n, m):
    return n // math.gcd(n, m) * m

gcd_ = math.gcd(a, b)
lcm_ = lcm(a, b)

print(gcd_)
print(lcm_)

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
const fs = require('fs')
const [a, b] = fs.readFileSync(0, 'utf8').trim().split(' ')

const gcd = (a, b) => {
    if (b > 0) {
        return gcd(b, a % b)
    } else {
        return a
    }
}

const lcm = (a, b) => {
    return a * b / gcd(a, b)
}

console.log(gcd(a, b))
console.log(lcm(a, b))